src/service/weatherData.js

```diff
@@ -0,0 +1,26 @@
+import Config from 'react-native-config';
```

Comment:
I would suggest that file is named just `weather.js` (same for `city.js`). `data` suffix is quite unnecessary: we are in a folder named `services` so we know it's some kind of service, we just need to know for which resource it is specified. In this case, that would be `weather`, giving us `weather.js`.